chat.go

```diff
@@ -46,6 +46,17 @@ type PostMessageParameters struct {
 	EscapeText      bool         `json:"escape_text"`
 }
 
+// PostEphemeralParameters contains all necessary params for sending chat.postEphemeral
+type PostEphemeralParameters struct {
+	Channel     string       `json:"channel"`
+	Text        string       `json:"text"`
+	User        string       `json:"user"`
+	AsUser      bool         `json:"as_user"`
+	Attachments []Attachment `json:"attachments"`
+	LinkNames   int          `json:"link_names"`
```

Comment:
`link_names` has a boolean type - https://api.slack.com/methods/chat.postEphemeral

Reply:
Hmm... It looks like in the previous version of API this property was an integer. Don't mind about my comment.